### PR TITLE
Css three column layout

### PR DESCRIPTION
**Pull request by Eugenepark215**
Screen Shot 2022-08-25 at 2 33 51 PM

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, looking pretty good so far, but we can improve the structure of the HTML from here. Instead of putting `display: flex` on your `.container`, move it to `.row` your basic structure should be something like this. One container, one row, and three columns.

```html
<div class="container">
  <div class="row">
    <div class="column">
      <-- image, heading, and paragraph -->
    </div>
    <div class="column">
      <-- image, heading, and paragraph -->
    </div>
    <div class="column">
      <-- image, heading, and paragraph -->
    </div>
  </div>
</div>
```

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-25 at 3 32 24 PM

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -25,8 +24,10 @@ h1 {
   font-weight: lighter;
 }
 
-.column {
+.row {
   background-color: none;
+  display: flex;
+  flex-basis: calc(100%/3);
```
Comment (thebearingedge) [inline css-three-column-layout/styles.css:30]: flex-basis only goes on flex items, so you can move this to your `.column` rule set.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -25,8 +24,10 @@ h1 {
...
 }
 
 section {
```
Comment (thebearingedge) [inline css-three-column-layout/styles.css:33]: To get the images dead center in the sections, you can use flexbox (see css-flexbox/align-items.html)

**Review by thebearingedge — CHANGES_REQUESTED** [review]
ok, the html is waaaayy, better. we just need some tweaks to the css. no rush

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-25 at 4 30 29 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**